edge-hub/src/Microsoft.Azure.Devices.Edge.Hub.MqttBrokerAdapter/ISubscriber.cs

```diff
@@ -0,0 +1,10 @@
+// Copyright (c) Microsoft. All rights reserved.
+namespace Microsoft.Azure.Devices.Edge.Hub.MqttBrokerAdapter
+{
+    using System.Collections.Generic;
+
+    public interface ISubscriber
```

Comment:
What is difference between a ISubscriber and IMessageConsumer?

Reply:
Logically it is two separate things to decide what messages should come from the broker and who should process those messages. One example was for that the 'subscription change' which was processed by multiple consumers, but there was a separate subscriber for the messages itself.
With the introduction of subscription watcher, this separation doesn't seem important right now, and the two interfaces could be merges, as right now the same class processes the message (consumes) than it subscribed to it.
I am not sure if it will not be handy to have the separation later.

Reply:
At the end, merged ISubscriber into IMessageConsumer